packages/core/src/components/button/button-vars.scss

```diff
@@ -65,17 +63,17 @@
   --tds-btn-ghost-border-color: transparent;
   --tds-btn-ghost-background-hover: transparent;
   --tds-btn-ghost-color-hover: var(--tds-black);
-  --tds-btn-ghost-border-color-hover: rgb(0 0 0 / 60%);
-  --tds-btn-ghost-outline-color-hover: rgb(0 0 0 / 60%);
+  --tds-btn-ghost-border-color-hover: var(--tds-black-48);
+  --tds-btn-ghost-outline-color-hover: var(--tds-black-48);
   --tds-btn-ghost-background-active: transparent;
   --tds-btn-ghost-color-active: var(--tds-black);
-  --tds-btn-ghost-border-color-active: rgb(0 0 0 / 87%);
+  --tds-btn-ghost-border-color-active: var(--tds-black-87);
   --tds-btn-ghost-background-focus: transparent;
   --tds-btn-ghost-color-focus: var(--tds-black);
   --tds-btn-ghost-border-color-focus: var(--tds-blue-300);
   --tds-btn-ghost-outline-color-focus: var(--tds-blue-300);
```

Comment:
In Figma the outline color for the ghost button when focused is Blue 400 but here it is Blue 300. Which is correct?
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/588898ec-3e48-41ac-ac08-a2d368e1c9b9)

Reply:
Nice catch, I have updated it as per Figma file ;)